api/pkg/apis/v1alpha1/providers/config/catalog/catalogprovider.go

```diff
@@ -87,21 +88,23 @@ func CatalogConfigProviderConfigFromMap(properties map[string]string) (CatalogCo
 	ret.Password = password
 	return ret, nil
 }
-func (m *CatalogConfigProvider) unwindOverrides(override string, field string) (string, error) {
-	catalog, err := utils.GetCatalog(context.TODO(), m.Config.BaseUrl, override, m.Config.User, m.Config.Password)
+func (m *CatalogConfigProvider) unwindOverrides(override string, field string, scope string) (string, error) {
+	catalog, err := utils.GetCatalog(context.TODO(), m.Config.BaseUrl, override, m.Config.User, m.Config.Password, scope)
```

Comment:
This means catalog inheritance must happen in same scope

Reply:
Correct